src/main/java/seedu/address/model/fields/Outcome.java

```diff
@@ -0,0 +1,26 @@
+package seedu.address.model.fields;
+
+/**
+ * Represents the possible outcome for an application/task
+ */
+public enum Outcome {
+    FOLLOW_UP, GHOSTED, REJECTED, OFFERED, ACCEPTED;
+
+    @Override
+    public String toString() {
+        switch (this) {
```

Comment:
maybe switch (this.toLowerCase())?

Reply:
FOLLOW_UP got underscore, cannot directly covert to Follow-up, abit laze, so i just handle all cases

Reply:
mb I was thinking of the parser, but for the returns can capitalise first letter and `Follow-up` here as well?